src/IO/AssimpLoader/AssimpGeometryDataLoader.cpp

```diff
@@ -174,7 +174,7 @@ void AssimpGeometryDataLoader::fetchVertices( const aiMesh& mesh, GeometryData&
     const int size = mesh.mNumVertices;
     auto& vertex   = data.getVertices();
     vertex.resize( mesh.mNumVertices );
-#pragma omp parallel for
+    //#pragma omp parallel for
```

Comment:
why commenting out this pragma ?

Reply:
should not, I revert this one (I have commented to test if it removes omp errors with valgrind)